data/ecosystems/s/starknet.toml

```diff
@@ -2806,9 +2807,21 @@ url = "https://github.com/MusebotAi/starknet"
 [[repo]]
 url = "https://github.com/myBraavos/braavos-account-cairo"
 
+[[repo]]
+url = "https://github.com/myBraavos/efficient-secp256r1"
+
+[[repo]]
+url = "https://github.com/myBraavos/starknet.cc"
+
+[[repo]]
+url = "https://github.com/myBraavos/starknet-dapp"
+
 [[repo]]
 url = "https://github.com/myBraavos/starknet-basecamp-aa-example"
 
+[[repo]]
+url = "https://github.com/myBraavos/starkhouse-execute-deepdive"
+
```

Comment:
If it's not a problem, then I guess it's better to list it here as well, just in case someone parses the ecosystem directly without delving into sub-ecosystems